Neos.Neos/Resources/Private/Fusion/Prototypes/DimensionsMenuItems.fusion

```diff
@@ -0,0 +1,17 @@
+prototype(Neos.Neos:DimensionsMenuItems) < prototype(Neos.Neos:MenuItems)  {
+ @class = 'Neos\\Neos\\Fusion\\DimensionsMenuItemsImplementation'
+
+	node = ${node}
```

Comment:
This is already defined in `Neos.Neos:MenuItems` - do we still need it here?

Reply:
No i removed it.